data/tools/wmllint

```diff
@@ -2884,10 +2884,12 @@ def translator(filename, mapxforms, textxform):
                     else:
                         fields = [x for x in line]
                     outmap.append(fields)
-                    if not maskwarn and maptype == 'map' and re.search('_s|_f(?!me)', line):
-                        print('"%s", line %d: warning, fog or shroud in map file' \
-                              % (filename, lineno+1))
-                        maskwarn = True
+                    terrain_strings = (f.split()[-1] for f in fields)
```

Comment:
I think this line should be moved inside the outermost conditional

Reply:
It took me way too long to figure out which conditional you are referring to as "outermost". I though you meant this one
https://github.com/wesnoth/wesnoth/blob/6a0baa563f8b3a6aede975debd657e6402ec2af5/data/tools/wmllint#L2829
and was trying to understand why or how. But now I see that you were referring to
```py
if not maskwarn and maptype == 'map':
```

Reply:
Done